src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -108,8 +108,9 @@ function getLandoConfig() {
 			'@lando/wordpress',
 		],
 		proxyName: 'vip-dev-env-proxy',
-		userConfRoot: getLandoUserConfigurationRoot(),
-		home: '',
+		userConfRoot: landoDir,
+		home: fakeHomeDir,
+		domain: 'lndo.site',
```

Comment:
The `domain` key is used to name the certificate file; if we don't pass it, we'll get `undefined.pem`, which is a bit confusing.